HW1/P2/P2b.py

```diff
@@ -1,3 +1,31 @@
 from P2 import *
+import matplotlib.pyplot as plt
+from pyspark import SparkContext
+import random
 
-# Your code here
+# constants
+GRID_DIM = 2000
+PART_DIM = 10
+PLOT_BINS = 20
+
+sc = SparkContext("local", "Mandelbrot load balanced")
+
+# randomize coordinates first, then partition as in default
+rand_coords = range(GRID_DIM)
+random.shuffle(rand_coords)
+dim_1 = sc.parallelize(rand_coords, PART_DIM)
+grid = dim_1.cartesian(dim_1)
```

Comment:
I am not sure how the cartesian function does things, but I got a better shuffle results when I call "shuffle(x_y_coordinates)" directly (ie not using the cartesian)